winit-core/src/window.rs

```diff
@@ -19,9 +19,9 @@ use crate::monitor::{Fullscreen, MonitorHandle};
 /// Whenever you receive an event specific to a window, this event contains a `WindowId` which you
 /// can then compare to the ids of your windows.
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct WindowId(usize);
+pub struct SurfaceId(usize);
 
-impl WindowId {
+impl SurfaceId {
```

Comment:
Drop rename and just add an alias with the name `SurfaceId`.

Reply:
Why not make `WindowId` the alias?